packages/core/contracts/chainbridge/Bridge.sol

```diff
@@ -0,0 +1,452 @@
+pragma solidity ^0.6.0;
```

Comment:
I really wish we didn't have to copy these contracts in, but that seems to be the only thing we can do since their package doesn't export the solidity, only the typescript bindings and ethers objects :/